data/strings/sound.txt

```diff
@@ -2525,7 +2525,7 @@
     pt-BR = Radar à frente
     ro = Cameră frontală
     ru = Впереди камера
-    sk = Kamera vpredu
+    sk = Pozor, radar!
```

Comment:
Isn't it confusing to notify users about a radar instead of a camera?

Reply:
Well, a speed camera is equipped with a doppler radar to measure the speed, so technically it is correct... However, the main reason is that in Slovakia, it is more common to call these devices "radars", therefore I think it will be less confusing this way. Actually, we are not the only ones who call it that way - I see "radar" mentioned in czech, polish and brazilian translations as well.
I guess it goes back to times when the police used manually operated speed radars which only displayed the speed to the officer. Addition of an automatic camera is a fairly new improvement, and we still stick to the "old" name :)

Reply:
I see your point. There are also cameras that take photos on red light and on wrong turns. Ok, let's give it a go.